src/features/applications/hooks/slug.ts

```diff
@@ -59,7 +59,7 @@ hooks.addPureHook('PATCH', 'resin', 'application', {
 						id: { $in: ids },
 					},
 				},
-			});
+			} as const);
```

Comment:
I am not sure what is the best pattern here - 
1 - add const on the whole object as we know we are not changing it
2 - add const only where it is necessary (in this case, just the `organization: { ... } as const`)

No blockers on this, just wondering which is best